airbyte-scheduler/scheduler-persistence/src/main/java/io/airbyte/scheduler/persistence/DefaultJobPersistence.java

```diff
@@ -373,6 +406,22 @@ private static RecordMapper<Record, SyncStats> getSyncStatsRecordMapper() {
         .withMaxSecondsBetweenStateMessageEmittedandCommitted(record.get(SYNC_STATS.MAX_SECONDS_BETWEEN_STATE_MESSAGE_EMITTED_AND_COMMITTED));
   }
 
+  private static RecordMapper<Record, NormalizationSummary> getNormalizationSummaryRecordMapper() {
+    final ObjectMapper mapper = new ObjectMapper();
+    return record -> {
+      try {
+        return new NormalizationSummary().withStartTime(record.get(NORMALIZATION_SUMMARIES.START_TIME).toInstant().toEpochMilli())
+            .withEndTime(record.get(NORMALIZATION_SUMMARIES.END_TIME).toInstant().toEpochMilli())
+            .withFailures(record.get(NORMALIZATION_SUMMARIES.FAILURES, String.class) == null ? null
+                : Lists.newArrayList(mapper.readValue(String.valueOf(record.get(NORMALIZATION_SUMMARIES.FAILURES)), FailureReason[].class)));
+      } catch (final JsonProcessingException e) {
+        return new NormalizationSummary().withStartTime(record.get(NORMALIZATION_SUMMARIES.START_TIME).toInstant().toEpochMilli())
```

Comment:
Suggestion:  add a comment (and possible a debug log statement for the exception) as to why we are returning this default normalization summary when the JSON exception occurs.

Reply:
I just added a comment - let me know what you think. I thought we should rescue this error since nobody is actually reading this field from the app so I don't want this read to cause an error.

Reply:
Should this also throw an exception in order to have real failure?

Reply:
I rescued the exception since nobody is reading this field from the app